sklearn/utils/estimator_checks.py

```diff
@@ -421,19 +423,18 @@ def _generate_sparse_matrix(X_csr):
     """
 
     for sparse_format in ['dok', 'lil', 'dia', 'bsr', 'csr', 'csc', 'coo']:
-        yield {"format": sparse_format, "X": X_csr.asformat(sparse_format)}
+        yield (sparse_format, X_csr.asformat(sparse_format))
+    if NumpyVersion(scipy_version) >= '0.14.0':
```

Comment:
NumpyVersion is not available, ironically, in all supported versions of numpy. Elsewhere we use `from distutils.version import LooseVersion`